From forest@alittletooquiet.net  Tue May 12 09:36:54 2009
From: Forest Bond <forest@alittletooquiet.net>
Date: Mon, 4 May 2009 21:48:27 -0400
Subject: Staging: vt6655: Build vt6655.ko, not viawget.ko.
To: Greg KH <greg@kroah.com>
Cc: Larry Finger <Larry.Finger@lwfinger.net>
Message-ID: <20090505014827.GW13354@storm.local.network>


As you requested, this series is to be applied on top of patches 1, 2, and 6
(and replaces patches 3, 4, 5, 7, and 8) from the previous series.

Build vt6655.ko, not viawget.ko.

Signed-off-by: Forest Bond <forest@alittletooquiet.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/vt6655/device_cfg.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/vt6655/device_cfg.h
+++ b/drivers/staging/vt6655/device_cfg.h
@@ -86,7 +86,7 @@ struct _version {
 #define VCAM_SIZE           32
 #define TX_QUEUE_NO         8
 
-#define DEVICE_NAME         "viawget"
+#define DEVICE_NAME         "vt6655"
 #define DEVICE_FULL_DRV_NAM "VIA Networking Solomon-A/B/G Wireless LAN Adapter Driver"
 
 #ifndef MAJOR_VERSION
